src/test/java/com/squareup/kotlinpoet/KotlinPoetTest.kt

```diff
@@ -310,7 +310,7 @@ class KotlinPoetTest {
         .addFunction(FunSpec.builder("shrink")
             .returns(String::class)
             .receiver(LambdaTypeName.get(
-                parameters = String::class.asClassName(),
+                parameters = *arrayOf(String::class.asClassName()),
```

Comment:
Well this is gross...

Reply:
Passing varargs as named parameters is deprecated now, not sure why, I'll try to find more info. Unfortunately `[]` don't work here.